web/e2e/homepage.spec.ts

```
@@ -0,0 +1,143 @@
+      const mapContainer = page.locator('[style*="height"]').first()
+      await expect(mapContainer).toBeVisible()
+    })
+
+    test('When the page loads, Then the navigation elements should be accessible', async ({ page }) => {
+      // Then the user menu button should be visible
+      const userMenuButton = page.getByRole('button', { name: 'ユーザーメニューを開く' })
+      await expect(userMenuButton).toBeVisible()
+      
+      // And desktop navigation should be visible on larger screens
+      if (page.viewportSize()?.width! >= 768) {
+        await expect(page.getByText('スポットを探す').first()).toBeVisible()
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)